eng/scripts/Setup-MQTTContainer.ps1

```diff
@@ -0,0 +1,29 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $AgentImage,
+    [string] $ConfigFile
+)
+
+if ($IsLinux -and $AgentImage -match "ubuntu") {
+    $dockerGpgDir = "/etc/apt/keyrings"
+    $dockerGpgPath = "/etc/apt/keyrings/docker.gpg"
+    $dockerGpgUrl = "https://download.docker.com/linux/ubuntu/gpg"
+    $dockerListPath = "/etc/apt/sources.list.d/docker.list"
+
+    sudo install -m 0755 -d $dockerGpgDir
+
+    sudo curl -fsSL $dockerGpgUrl | sudo gpg --dearmor -o $dockerGpgPath
+
+    sudo chmod a+r /etc/apt/keyrings/docker.gpg
+    
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=$($dockerGpgPath)] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee $dockerListPath > /dev/null
+
+    sudo apt-get update
+
+    sudo apt-get -y install docker-ce
+
+    docker pull eclipse-mosquitto
```

Comment:
I am currently pulling the container every time the test is run, @danieljurek will help with hosting it ourselves.